Models/post-and-patch-models.js

```
@@ -0,0 +1,30 @@
+const { copy } = require('../app.js')
+const db = require('../db/connection.js')
+
+exports.insertNewComment = (newComment, articleId, author) => {
+    
+      if(isNaN(articleId)){
+        return Promise.reject({ status: 400, msg: 'bad request' })
+      }
```

PSQL should throw an error if the articleID is not a valid number.